test/testdata/configs/config-v28.json

```diff
@@ -85,6 +85,9 @@
     "OptimizeAccountsDatabaseOnStartup": false,
     "OutgoingMessageFilterBucketCount": 3,
     "OutgoingMessageFilterBucketSize": 128,
+    "P2PEnable": false,
+    "P2PPersistPeerID": true,
+    "P2PPrivateKeyLocation": "",
```

Comment:
do we need to update this file?

Reply:
Ah yes, that looks leftover from the previous version. Good catch.